docs/UserGuide.md

```diff
@@ -357,20 +361,50 @@ Apart from supporting the core HR functions, Coydir also supports *department-le
 With these features, we offer a view that is broader than an individual employee profile and more specific than the full directory. 
 Complete with **real-time statistics and analytics**, Coydir makes department management much simpler.
 
-**[Insert details on how we have fixed departments for now]**
+Currently, our application only provides a list of default departments for the user to choose from. Future update will include customizable department name. Our current list of valid departments are:
+- Administration
+- Board of Directors
+- Customer Service
+- Finance
+- General Management
+- Human Resources
+- Information Technology
+- Legal
+- Marketing
+- Operations
+- Product Management
+- Production
+- Quality Assurance
+- Research and Development
+- Sales
+- Technology
 
 #### View details of a department: `viewdepartment`
 
 View the summarized details of a department given the name of the department.
```

Comment:
Can explain a bit more on why we need `viewdepartment`. Maybe can say that as a department head, you will need to track the ratings of your employees for talent development and see how many people are in for manpower allocation